### PR TITLE
Add audit sections for the tests with TODO markers, #259

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParams2Test.java [hunk 1]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class EnhancementMultipleParams2Test extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementMultipleParamsExtension2.class)
-                                      .withClass(EnhancementMultipleParamsExtension2.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementMultipleParamsExtension2.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParams2Test.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParamsTest.java [hunk 2]
```diff
@@ -3,17 +3,22 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class EnhancementMultipleParamsTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementMultipleParamsExtension.class)
-                                      .withClass(EnhancementMultipleParamsExtension.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementMultipleParamsExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParamsTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementNoParamTest.java [hunk 3]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class EnhancementNoParamTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementNoParamExtension.class)
-                                      .withClass(EnhancementNoParamExtension.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementNoParamExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementNoParamTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementOnlyMessagesTest.java [hunk 4]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class EnhancementOnlyMessagesTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementOnlyMessagesExtension.class)
-                                      .withClass(EnhancementOnlyMessagesExtension.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementOnlyMessagesExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementOnlyMessagesTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementOnlyTypesTest.java [hunk 5]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class EnhancementOnlyTypesTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementOnlyTypesExtension.class)
-                                      .withClass(EnhancementOnlyTypesExtension.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(EnhancementOnlyTypesExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementOnlyTypesTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationMultipleParams2Test.java [hunk 6]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class RegistrationMultipleParams2Test extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationMultipleParamsExtension2.class)
-                                      .withClass(RegistrationMultipleParamsExtension2.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationMultipleParamsExtension2.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationMultipleParams2Test.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationMultipleParamsTest.java [hunk 7]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class RegistrationMultipleParamsTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationMultipleParamsExtension.class)
-                                      .withClass(RegistrationMultipleParamsExtension.class)
-                                      .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationMultipleParamsExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationMultipleParamsTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationNoParamTest.java [hunk 8]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class RegistrationNoParamTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationNoParamExtension.class)
-                        .withClass(RegistrationNoParamExtension.class)
-                        .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationNoParamExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationNoParamTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationOnlyMessagesTest.java [hunk 9]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class RegistrationOnlyMessagesTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationOnlyMessagesExtension.class)
-                        .withClass(RegistrationOnlyMessagesExtension.class)
-                        .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationOnlyMessagesExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationOnlyMessagesTest.java:18]: This reverts a fix from #360.

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationOnlyTypesTest.java [hunk 10]
```diff
@@ -3,17 +3,23 @@
...
 @SpecVersion(spec = "cdi", version = "4.0")
 public class RegistrationOnlyTypesTest extends AbstractInvalidExtensionParamTest {
 
     @ShouldThrowException(DefinitionException.class)
     @Deployment
     public static WebArchive createTestArchive() {
-        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationOnlyTypesExtension.class)
-                        .withClass(RegistrationOnlyTypesExtension.class)
-                        .build();
+        return prepareArchiveBuilder().withBuildCompatibleExtension(RegistrationOnlyTypesExtension.class).build();
```
Comment (Ladicek) [inline impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/RegistrationOnlyTypesTest.java:18]: This reverts a fix from #360.

**Comment by Ladicek** [comment]
Heroic!

Hats Off!

File: impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParams2Test.java (hunk 1 above)
Comment (starksm64) [reply impl/src/main/java/org/jboss/cdi/tck/tests/build/compatible/extensions/invalid/EnhancementMultipleParams2Test.java:18]: Ok, let me fix these.

**Review by Ladicek — APPROVED** [review]
LGTM!

**Review by manovotn — APPROVED** [review]
Awesome, thanks @starksm64!

**Pull request merged**